<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyContactDataClass</class>
 <widget class="QDialog" name="CompanyContactDataClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompanyContactData</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lb_quickSearch">
         <property name="text">
          <string>Quick Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_quickSearch"/>
       </item>
       <item>
        <widget class="QWidget" name="advancedPlaceholder" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pb_search">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_reset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tv_company"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
